### PR TITLE
feat: Add some conversion methods to AccountIdentifier in ic-ledger-types

### PR DESCRIPTION
**Pull request by witter-deland**
# Description

Add some conversion methods to AccountIdentifier for better conversion between types

Fixes # (issue)

# How Has This Been Tested?

Code from the already tested [ic repository](https://github.com/dfinity/ic/blob/c30d4b9f4459adcaf06b84328e44896155a45a5f/rs/rosetta-api/icp_ledger/src/account_identifier.rs)

# Checklist:

- [x] The title of this PR complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] I have edited the CHANGELOG accordingly.
- [x] I have made corresponding changes to the documentation.

### CODE REVIEW

**Comment by roman-kashitsyn** [comment]
If we're copying some code, it would be great to copy the test suite for it as well :)

**Comment by witter-deland** [comment]
Good idea
> If we're copying some code, it would be great to copy the test suite for it as well :)

File: library/ic-ledger-types/src/lib.rs [hunk 1]
```diff
@@ -9,16 +9,18 @@
     clippy::missing_safety_doc
 )]
 
+use std::convert::TryFrom;
+use std::fmt::{self, Display, Formatter};
+use std::ops::{Add, AddAssign, Sub, SubAssign};
+
```
Comment (roman-kashitsyn) [inline library/ic-ledger-types/src/lib.rs:15]: Could we get rid of blank lines between imports?

**Review by roman-kashitsyn — APPROVED** [review]
LGTM

File: library/ic-ledger-types/src/lib.rs (hunk 1 above)
Comment (witter-deland) [reply library/ic-ledger-types/src/lib.rs:15]: We use cargo fmt as the default code formatting tool for rust, cargo fmt is such a formatting style

Comment (witter-deland) [reply library/ic-ledger-types/src/lib.rs:15]: Of course, if the formatting style of the official team is different from cargo fmt, please let me know, and I will delete this blank line

**Pull request merged**